chromiumcontent/BUILD.gn

```diff
@@ -465,15 +427,12 @@ static_library("pepper_flash") {
     "//chrome/renderer/pepper/pepper_shared_memory_message_filter.cc",
     "//chrome/renderer/pepper/pepper_shared_memory_message_filter.h",
   ]
-  deps = [
-    "//media:media_features",
-    "//ppapi/host",
-    "//ppapi/proxy",
-    "//ppapi/proxy:ipc",
-    "//ppapi/shared_impl",
+  data_deps = [
```

Comment:
it's surprising to me that mojom and flapper turn into data_deps here. can you explain why this change is needed?

Reply:
No specific reason, I was just testing pepper flash without linking those. Sorry for the mislead here.